protocols/gossipsub/src/lib.rs

```diff
@@ -142,7 +142,7 @@ pub mod protocol;
 
 mod backoff;
 mod behaviour;
-mod config;
+pub mod config;
```

Comment:
not sure if that should be public...

Reply:
I'd rather not, what prompted the change?

Reply:
If I remember correctly, the `::config::` prefix. I've reverted to private.